index.html

```diff
@@ -1,11 +1,167 @@
+        <li>
+          <h3 class="features__title">
+            Пунктуальність
+          </h3>
+          <p class="features__text">
+            Так само консультація з широким активом значною мірою зумовлює.
+          </p>
+        </li>
+        <li>
+          <h3 class="features__title">
+            Сучасні технології
+          </h3>
+          <p class="features__text">
+            Значимість цих проблем настільки очевидна, що реалізація планових завдань.
+          </p>
+        </li>
+      </ul>
+    </section>
+
+    <section class="services">
+      <h2 class="services__title">
+        Чим ми займаємося
+      </h2>
+      <ul class="services__list">
+        <li>
+          <img src="images/Services/keyboard.jpg" alt="Чим мі займаємося" width="370" height="294">
+        </li>
+        <li>
+          <img src="images/Services/phone.jpg" alt="Чим мі займаємося" width="370" height="294">
+        </li>
+        <li>
+          <img src="images/Services/rpb.jpg" alt="Чим мі займаємося" width="370" height="294">
+          <!--Изображения выше имеют нужное разрешение, поэтому ни ширину, ни высоту менять не стал-->
+        </li>
+      </ul>
+    </section>
+
+    <section class="team">
+      <h2 class="team__title services__title">
+        Наша команда
+      </h2>
+      <ul class="team__list">
+        <li>
+          <img src="images/Team/Ihor_Demyanenko.jpg" alt="Ігор Дем'яненко" width="270" height="260">
+
+          <h3 class="team__name">
+            Ігор Дем'яненко
+          </h3>
+          <p class="team__text">
+            Product Designer
+          </p>
+        </li>
+        <li>
+          <img src="images/Team/Olga_Repina.jpg" alt="Ольга Рєпіна" width="270" height="260">
+          <h3 class="team__name ">
+            Ольга Рєпіна
+          </h3>
+          <p class="team__text">
+            Frontend Developer
+          </p>
+        </li>
+        <li>
+          <img src="images/Team/Mykola_Tarasov.jpg" alt="Микола Тарасов" width="270" height="260">
+          <h3 class="team__name">
+            Микола Тарасов
+          </h3>
+          <p class="team__text">
+            Marketing
+          </p>
+        </li>
+        <li>
+          <img src="images/Team/Mykhailo_Yermakov.jpg" alt="Михайло Ермаков" width="270" height="260">
+          <h3 class="team__name">
+            Михайло Ермаков
+          </h3>
+          <p class="team__text">
+            UI Designer
+          </p>
+        </li>
+      </ul>
+    </section>
+  </main>
+
+  <footer class="footer">
+    <a class="logo" href="index.html">
+      <span class="logo__web">Web
+      </span>
+      <span class="logo__studio logo__studio--white">Studio
+      </span>
+    </a>
+    <address>
+      м. Київ, пр-т Лесі Українки, 26 <br>
+      <a class="contacts__link--color" href="mailto:info@devstudio.com">info@devstudio.com</a> <br>
+      <a class="contacts__link--color" href="tel:+380961111111">+38 096 111 11 11</a>
+    </address>
```

Comment:
tag `<address>` should only contain the address, not email and phone contacts

```suggestion
    <address>
      м. Київ, пр-т Лесі Українки, 26 <br>
    </address>

    <a class="contacts__link--color" href="mailto:info@devstudio.com">info@devstudio.com</a> <br>
    <a class="contacts__link--color" href="tel:+380961111111">+38 096 111 11 11</a>
```